<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriangleForm</class>
 <widget class="QWidget" name="TriangleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>三角形参数</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TrianglePointForm" name="mPointForm1" native="true"/>
   </item>
   <item>
    <widget class="TrianglePointForm" name="mPointForm2" native="true"/>
   </item>
   <item>
    <widget class="TrianglePointForm" name="mPointForm3" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TrianglePointForm</class>
   <extends>QWidget</extends>
   <header>trianglepointform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
